tap_pendo/streams.py

```diff
@@ -1000,7 +1001,8 @@ class Visitors(LazyAggregationStream):
     key_properties = ["visitor_id"]
 
     def get_body(self):
-        include_anonymous_visitors = bool(self.config.get('include_anonymous_visitors', 'false').lower() == 'true')
+        include_anonymous_visitors = self.config.get('include_anonymous_visitors', 'False') or DEFAULT_INCLUDE_ANONYMOUS_VISITORS
+        anons = include_anonymous_visitors == 'true'
```

Comment:
These lines are a little tricky to reason about and I think we can do better.

I have two issues with this:
1. We use a `.get()` and fall back to a random string
2. `DEFAULT_INCLUDE_ANONYMOUS_VISITORS` is a boolean that will get compared to a string `"true"`.
3. We've introduced a breaking change where a config with `"TRUE"` or `"True"` (or any other spelling) returns the wrong value now

---

Issue 1 is a problem because any string could go in the fallback and this function would behave the same.
Right now, the things that come out of this `get()` are
- `"true"`
- `"false"`
- any other string the user could put into the config
- `"False"`, or our fallback
- `None`

Out of all of these, `None` is the only one that will cause the second fallback to `DEFAULT_INCLUDE_ANONYMOUS_VISITORS` to happen. The rest of the values are converted to booleans on Line 1005.

---

Issue 2 is a problem because it's always false. 
```python
>>> False == 'true'
False
>>> True == 'true'
False
```

While that's logically sound in this case, it feels like an accident that we are leaning on this. This feels like a personal style thing to not want to compare apples to oranges, but I think it's important to call out.

---

Issue 3 is a problem because by using the `.lower()` in the old implementation, we decided to support all of the capitalizations of "true". So this PR would break that and we shouldn't break that

Reply:
For Issue 1, I think it's better if we cut down the possible values from

- "true"
- "false"
- any other string the user could put into the config
- "False", or our fallback
- None

to

- "true"
- "false"
- any other string the user could put into the config
- None